### PR TITLE
CB-16901 The CM stop is skipping only if the CM status timed out, thi…

### PR DESCRIPTION
**Pull request by topolyai5**
…s can cause a long datahub termination. In this PR we stop the CM if the status is success

See detailed description in the commit message.

### CODE REVIEW

File: cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java [hunk 1]
```diff
@@ -828,10 +828,17 @@ public void stopCluster(boolean disableKnoxAutorestart) throws CloudbreakExcepti
...
+            if (extendedPollingResult.isSuccess()) {
                 stopWithRunningCm(disableKnoxAutorestart, cluster, clustersResourceApi);
+            } else {
+                String errorMessage = "Unknown error from poller";
+                if (extendedPollingResult.getException() != null) {
+                    errorMessage = extendedPollingResult.getException().getMessage();
+                }
+                LOGGER.info("We will skip the CM stop, because the polling result wasn't success: {}, error: {}", extendedPollingResult.getPollingResult(),
+                        errorMessage, extendedPollingResult.getException());
+                skipStopWithStoppedCm();
```
Comment (pdarvasi) [inline cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java:841]: nit: could you pls extract this logic to a private method? thanks

Comment (topolyai5) [reply cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java]: done

**Review by pdarvasi — APPROVED** [review]

**Comment by pdarvasi** [comment]
@topolyai5 Pls check the IT so that I could merge the PR, thanks!

**Comment by topolyai5** [comment]
retest this please

**Pull request merged**